tests/Form/InputUrlTest.php

```
@@ -0,0 +1,610 @@
+        self::assertSame(
+            <<<HTML
+            <input id="inputurl-" type="url" form="form-id">
+            HTML,
+            InputUrl::tag()->form('form-id')->id('inputurl-')->render(),
+            "Failed asserting that element renders correctly with 'form' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(InputUrl::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)